app/views/edit/project.html

```diff
@@ -0,0 +1,53 @@
+<default-header class="top-header"></default-header>
+<div class="wrap no-sidebar">
+  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
+    <navbar-utility-mobile></navbar-utility-mobile>
+  </div>
+  <div class="middle">
+    <!-- Middle section -->
+    <div class="middle-section surface-shaded">
+      <div class="middle-container has-scroll">
+        <div class="middle-content">
+          <div class="container surface-shaded gutter-top">
+            <div class="col-md-12">
+              <h1>Edit Project {{project.metadata.name}}</h1>
+              <div class="help-block mar-bottom-lg" style="margin-top: -15px;">Update the descriptive information about your project.  The project's unique name can not be modified.</div>
```

Comment:
Maybe "Update the display name and description of your project. The project's unique name cannot be modified."